src/amo/components/Page/index.js

```diff
@@ -0,0 +1,60 @@
+/* @flow */
+import makeClassName from 'classnames';
+import config from 'config';
+import * as React from 'react';
+import { withRouter } from 'react-router-dom';
+
+import AppBanner from 'amo/components/AppBanner';
+import Footer from 'amo/components/Footer';
+import Header from 'amo/components/Header';
+import InfoDialog from 'core/components/InfoDialog';
+import type { ReactRouterLocationType } from 'core/types/router';
+
+import './styles.scss';
+
+type Props = {|
+  children: React.Node,
+  isHomePage?: boolean,
+|};
+
+type InternalProps = {|
+  ...Props,
+  _config: typeof config,
+  location: ReactRouterLocationType,
+|};
+
+export const PageBase = ({
+  _config = config,
+  children,
+  isHomePage = false,
+  location,
+}: InternalProps) => {
+  return (
+    <div className="App-amo">
+      <InfoDialog />
+
+      <Header isHomePage={isHomePage} location={location} />
+
+      <div className="App-content">
```

Comment:
Has the style moved to `Page/styles.scss` too?

Reply:
I'll move them